### PR TITLE
(DOCSP-13196) Stitch Deprecation doc - IOS

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-13196

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-13196-stitchtorealm-ios/ios/migrate

### CODE REVIEW

File: source/ios/migrate.txt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+--------
+If you have an existing app built with the Stitch SDK, you should migrate your
+app to use the new Realm SDK.  While much of the application logic and flow of
+information hasn't changed, there are a few changes to how the SDK connects to the
+{+realm+} backend.
+
+New Features
+------------
+
+- The {+service-short+} IOS SDK supports IOS applications written in Objective C.
```
Comment (cbush) [inline source/ios/migrate.txt:25]: It is iOS, not IOS.

Comment (cbush) [inline source/ios/migrate.txt:25]:
```suggestion
- The {+service-short+} IOS SDK supports IOS applications written in Objective-C.
```

Comment (cbush) [reply source/ios/migrate.txt:25]: And Swift.

File: source/ios/migrate.txt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+     - Rewrite the application logic to log out using the ``User`` object's ``logOut()`` method.
+
+   * - The Stitch SDK defined credentials through functions named after the credential type, i.e., ``AnonymousCredential()`` for an `anonymous credential <http://stitch-sdks.s3-website-us-east-1.amazonaws.com/stitch-sdks/swift/6/Structs/AnonymousCredential.html>`__. The Realm SDK has a class, :swift-sdk:`Credentials <Typealiases.html#/s:10RealmSwift11Credentialsa>`, with methods or enums for each credential type, i.e., ``Credentials.emailPassword()``.
+     - Use the Realm SDK's ``Credentials`` class to create authentication credentials.
+ 
+   * - The API to call a {+service-short+} function has changed from `stitchAppClient.callFunction() <http://stitch-sdks.s3-website-us-east-1.amazonaws.com/stitch-sdks/swift/6/Protocols/StitchAppClient.html#/s:10StitchCore0A9AppClientP12callFunction8withName0G4Args_ySS_Say10MongoSwift9BSONValue_pGyAA0A6ResultOyqd__GctSeRd__lF>`__ to :swift-sdk:`app.currentUser?.functions.{myFunction}() <Extensions/User.html#/s:So7RLMUserC10RealmSwiftE9functionsAC9FunctionsVvp>`.
```
Comment (cbush) [inline source/ios/migrate.txt:57]: It might help to explicitly call out something like "where `{myFunction}` is the name of the function".

**Review by cbush — CHANGES_REQUESTED** [review]
Content looks good, but we can't publish "IOS". Please update to iOS.

File: source/ios/migrate.txt (hunk 1 above)
Comment (mongomoe) [reply source/ios/migrate.txt]: Fixed, in all areas where it was written "IOS"

Comment (mongomoe) [reply source/ios/migrate.txt]: The previous Stitch SDK was written in Swift as well. This is a new features section @cbush. It's also meant to parody the [Android version of this page](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/whatschanged/android/migrate#new-features)

Comment (mongomoe) [reply source/ios/migrate.txt]: Fixed the typo with "Objective C" -> "Objective-C" though

File: source/ios/migrate.txt (hunk 2 above)
Comment (mongomoe) [reply source/ios/migrate.txt]: 👍

**Review by cbush — APPROVED** [review]

**Pull request merged**